src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
```

Comment:
You should add an empty line before the method declaration.
Ref: https://mate-academy.github.io/style-guides/java/java.html#s4.6.1-vertical-whitespace